libjolie/src/jolie/lang/parse/OLVisitor.java

```diff
@@ -21,159 +21,168 @@
-	public void visit( LinkOutStatement n );
-	public void visit( AssignStatement n );
-	public void visit( AddAssignStatement n );
-	public void visit( SubtractAssignStatement n );
-	public void visit( MultiplyAssignStatement n );
-	public void visit( DivideAssignStatement n );
-	public void visit( IfStatement n );
-	public void visit( DefinitionCallStatement n );
-	public void visit( WhileStatement n );
-	public void visit( OrConditionNode n );
-	public void visit( AndConditionNode n );
-	public void visit( NotExpressionNode n );
-	public void visit( CompareConditionNode n );
-	public void visit( ConstantIntegerExpression n );
-	public void visit( ConstantDoubleExpression n );
-	public void visit( ConstantBoolExpression n );
-	public void visit( ConstantLongExpression n );
-	public void visit( ConstantStringExpression n );
-	public void visit( ProductExpressionNode n );
-	public void visit( SumExpressionNode n );
-	public void visit( VariableExpressionNode n );
-	public void visit( NullProcessStatement n );
-	public void visit( Scope n );
-	public void visit( InstallStatement n );
-	public void visit( CompensateStatement n );
-	public void visit( ThrowStatement n );
-	public void visit( ExitStatement n );
-	public void visit( ExecutionInfo n );
-	public void visit( CorrelationSetInfo n );
-	public void visit( InputPortInfo n );
-	public void visit( OutputPortInfo n );
-	public void visit( PointerStatement n );
-	public void visit( DeepCopyStatement n );
-	public void visit( RunStatement n );
-	public void visit( UndefStatement n );
-	public void visit( ValueVectorSizeExpressionNode n );
-	public void visit( PreIncrementStatement n );
-	public void visit( PostIncrementStatement n );
-	public void visit( PreDecrementStatement n );
-	public void visit( PostDecrementStatement n );
-	public void visit( ForStatement n );
-	public void visit( ForEachStatement n );
-	public void visit( SpawnStatement n );
-	public void visit( IsTypeExpressionNode n );
-	public void visit( InstanceOfExpressionNode n );
-	public void visit( TypeCastExpressionNode n );
-	public void visit( SynchronizedStatement n );
-	public void visit( CurrentHandlerStatement n );
-	public void visit( EmbeddedServiceNode n );
-	public void visit( InstallFixedVariableExpressionNode n );
-	public void visit( VariablePathNode n );
-	public void visit( TypeInlineDefinition n );
-	public void visit( TypeDefinitionLink n );
-	public void visit( InterfaceDefinition n );
-	public void visit( DocumentationComment n );
-	public void visit( FreshValueExpressionNode n );
-	public void visit( CourierDefinitionNode n );
-	public void visit( CourierChoiceStatement n );
-	public void visit( NotificationForwardStatement n );
-	public void visit( SolicitResponseForwardStatement n );
-	public void visit( InterfaceExtenderDefinition n );
-	public void visit( InlineTreeExpressionNode n );
-	public void visit( VoidExpressionNode n );
-	public void visit( ProvideUntilStatement n );
+public interface OLVisitor {
```

Comment:
Huh? The whole file changed but the change is actually minimal. Probably because of the different code styling? Can you use the code style we are currently using?